Sources/FluentDarkModeKitExample/AppDelegate.swift

```diff
@@ -17,17 +17,48 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
 
     DarkModeManager.register(with: application)
 
-    window = UIWindow()
-    window?.rootViewController = {
-      let tabBarController = UITabBarController()
-      tabBarController.viewControllers = [
-        NavigationController(rootViewController: MainViewController()),
-        NavigationController(rootViewController: ViewController()),
-      ]
-      return tabBarController
-    }()
+    if #available(iOS 13.0, *) {
+      return true
+    }
+
+    window = AppDelegate.createNewWindow()
     window?.makeKeyAndVisible()
 
     return true
   }
+
+  class func createNewWindow(with window: UIWindow) -> UIWindow {
```

Comment:
Question: should we use `static`? Which should we prefer?

Reply:
makes little difference here, but probably static is preferred because `static` func cannot be overridden in subclass whereas `class` func can

Reply:
IMO, it would be better to choose `static` over `class` whenever possible. I think Swift prefers `static`. There might be a best practice for this.

Reply:
let's go ahead with static here.